PendingReleaseNotes.md

```diff
@@ -3,4 +3,8 @@
 --------
 
 * fixed duplicated partitions & topics in fetch response #82 #89
+<<<<<<< HEAD
```

Comment:
merge conflict?

Reply:
Yeah, the file is just sitting there unsaved in my editor.